views/trading-stop.html

```diff
@@ -0,0 +1,21 @@
+{% extends "layout.html" %}
+{% block pageTitle %}
+    Trading status not supported
+{% endblock %}
+{% block backLink %}
+    {{ govukBackLink({
+text: "Back",
+href: "/check-trading-status"
```

Comment:
is this back link correct? Should it not be prefixed with /company/{number}?
Which means we'll likely have to pass the company number into the template

Reply:
I don't know I have not really thought about backlinks I'd prefer to deal with the separately.

Reply:
I don't think it's wise to move the story to ready for test if there is a known bug in the screen. You could fix in in a separate PR after this one, then the screen would be ready for test.

Reply:
It's not in the AC for the ticket and we are following the "little and often" approach I don't want to start bringing more stuff into the PRs as that is when they start to spriral out of control and become very large.

Reply:
That's why I suggested fixing it in another PR, stories can have multiple PRs. It seems to me like we are adding in bugs that in all likelihood someone else will have to fix.

Reply:
Can you remove the back link entirely from this pr?
And in a seperate pr add the correct back link in?